report_dynamic/views/report_dynamic_section.xml

```diff
@@ -0,0 +1,102 @@
+            <search string="Sections">
+                <field name="name" />
+                <field name="report_id" />
+                <filter
+                    name="Paragraph"
+                    string="Paragraph"
+                    domain="[('is_paragraph','=','True')]"
+                />
+            </search>
+        </field>
+    </record>
+    <record id="view_report_dynamic_section_tree" model="ir.ui.view">
+        <field name="name">report.dynamic.section.tree</field>
+        <field name="model">report.dynamic.section</field>
+        <field name="arch" type="xml">
+            <tree string="Report Section">
+                <field name="sequence" />
+                <field name="name" />
+                <field name="report_id" />
+                <field name="condition_python" />
+                <field name="condition_domain" />
+                <field name="condition_python_preview" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_report_dynamic_section_form" model="ir.ui.view">
+        <field name="name">report.dynamic.section.form</field>
+        <field name="model">report.dynamic.section</field>
+        <field name="arch" type="xml">
+            <form string="Report Section">
+                <sheet>
+                    <div class="oe_title">
+                        <label for="name" />
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                    </div>
+                    <div>
+                        <label for="report_id" />
+                        <field name="report_id" readonly="1" />
+                    </div>
+                    <group name="main">
+                        <group name="left">
+                            <notebook>
+                                <page name="content" string="Content">
+                                    <field name="content" widget="html" />
+                                    <div>
+                                        <label for="is_paragraph" />
+                                        <field name="is_paragraph" />
+                                    </div>
+                                </page>
+                                <page name="condition" string="Conditions">
+                                    <group>
+                                        <field name="condition_python" />
+                                        <field name="model_id_model" invisible="1" />
+                                        <field
+                                            name="condition_domain"
+                                            widget="domain"
+                                            options="{'model': 'model_id_model', 'in_dialog': True}"
+                                        />
+                                        <field name="condition_python_preview" />
+                                        <div>
```

Comment:
To make it more clear what the preview is generated for, show here a readonly related field to `resource_ref` of the parent calling it "Preview record"